PocketKit/Sources/PocketKit/Report/ReportRecommendationView.swift

```diff
@@ -73,17 +70,9 @@ struct ReportRecommendationView: View {
                 }
 
                 Button(action: submitReport) {
-                    HStack {
-                        Spacer()
-                        Text(isReported ? L10n.reported : L10n.submitFeedback)
-                            .style(Style.submitButtonStyle)
-                        Spacer()
-                    }
-                }
-                .frame(height: Constants.submitButtonHeight)
-                .tint(Constants.submitButtonTintColor)
-                .background(Constants.submitButtonBackgroundColor)
-                .cornerRadius(Constants.cornerRadius)
+                    Text(isReported ? L10n.reported : L10n.submitFeedback)
+                }.buttonStyle(PocketButtonStyle(.primary))
```

Comment:
we have an `ActionsPrimaryButtonStyle` that prob can be replaced, but out of scope for this